@here/harp-mapview/lib/MapView.ts

```diff
@@ -2392,6 +2395,23 @@ export class MapView extends THREE.EventDispatcher {
 
         this.m_movementDetector.checkCameraMoved(this, time);
```

Comment:
This is already returning true/false, so just move it into the if condition

Reply:
Thanks for noticing